ci/download-df.sh

```diff
@@ -16,40 +17,48 @@ cd "$DF_FOLDER/.."
 
 if [ -f receipt ]; then
     if [ "$selfmd5" != "$(cat receipt)" ]; then
-        echo "download-df.sh changed; removing DF"
+        echo "download-df.sh changed; re-downloading tarballs"
         rm receipt
     else
-        echo "Already downloaded $DF_VERSION"
+        echo "Already downloaded $DF_VERSION tarballs"
     fi
 fi
 
 if [ ! -f receipt ]; then
-    rm -f "$tardest"
+    rm -f "$df_tardest" "$save_tardest"
     minor=$(echo "$DF_VERSION" | cut -d. -f2)
     patch=$(echo "$DF_VERSION" | cut -d. -f3)
-    url="http://www.bay12games.com/dwarves/df_${minor}_${patch}_linux.tar.bz2"
-    echo Downloading
+    echo "Downloading DF $DF_VERSION"
     while read url; do
```

Comment:
I must be missing something, the url isn't initialized anymore.. or is it?

Reply:
> So is it the change in run-tests.py that enables fortress tests? I'm very interested to understand how we can run tests under fortress mode.

fortress mode tests have been possible for some time now. they're enabled by default when running `test` locally. If a test has `config.mode = 'fortress'` then the test harness will make sure there is a fort loaded before running the test.

The change to `run-tests.py` removes the parameter I put in to skip fortress-mode tests for CI, since the CI environment didn't have a fort save to use. This PR provides that fort save.

Reply:
> I must be missing something, the url isn't initialized anymore.. or is it?

the `url` variable on line 30 was never used. It's probably left over from a previous implementation before the while loop was added. the `url` on line 32 is initialized by the while loop.

Reply:
> `url` on line 32 is initialized by the while loop

ah, maybe I should bother myself to learn basic python =S seemed like it was parsing the url

> If a test has `config.mode = 'fortress'`

cool, thanks

Reply:
yeah, it can be difficult to read. note that the `download-df.sh` file is shell script (bash), though, not python.